Libraries/NativeAnimation/Drivers/RCTFrameAnimation.m

```diff
@@ -46,13 +48,17 @@ - (instancetype)initWithId:(NSNumber *)animationId
   if ((self = [super init])) {
     NSNumber *toValue = [RCTConvert NSNumber:config[@"toValue"]] ?: @1;
     NSArray<NSNumber *> *frames = [RCTConvert NSNumberArray:config[@"frames"]];
+    NSNumber *iterations = [RCTConvert NSNumber:config[@"iterations"]] ?: @1;
 
     _animationId = animationId;
     _toValue = toValue.floatValue;
     _fromValue = valueNode.value;
     _valueNode = valueNode;
     _frames = [frames copy];
     _callback = [callback copy];
+    _animationHasFinished = iterations.integerValue == 0;
```

Comment:
It is useful in case someone wants to pass iterations = 0 so the animation doesn't run. Doesn't seems like a useful thing to do but meh...